app/uk/gov/hmrc/customerprofile/connector/ApplePassConnector.scala

```diff
@@ -21,15 +21,16 @@ import com.google.inject.name.Named
 import play.api.Logger
 import play.api.http.Status.OK
 import play.api.libs.json.Format.GenericFormat
-import play.api.libs.json.{JsValue, Json}
+import play.api.libs.json.Json
 import uk.gov.hmrc.customerprofile.domain.{ApplePassIdGenerator, RetrieveApplePass}
-import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpException, HttpResponse}
+import uk.gov.hmrc.http.{HeaderCarrier, HttpException, HttpResponse, StringContextOps}
 import uk.gov.hmrc.http.HttpReads.Implicits._
+import uk.gov.hmrc.http.client.HttpClientV2
 
 import scala.concurrent.{ExecutionContext, Future}
 
 class ApplePassConnector @Inject() (
-  http:                                                          HttpClient,
+                                     http: HttpClientV2,
```

Comment:
The formatting here is off, and appears to be for the majority of the files that have been edited. If you need any help getting the formatter configured on your machine give me a shout